api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -2550,13 +2572,19 @@ public void saveConcept_shouldSetAuditInfoIfAnItemIsRemovedFromAnyOfItsChildColl
 		Concept concept = conceptService.getConcept(3);
 		Assert.assertNull(concept.getDateChanged());
 		Assert.assertNull(concept.getChangedBy());
-		
-		ConceptDescription description = concept.getDescription();
-		Assert.assertNotNull(description);
-		Assert.assertTrue(concept.removeDescription(description));
+
+		Concept concept1= conceptService.getConcept(5);
+		ConceptAnswer conceptanswer = new ConceptAnswer(concept1);
+		concept.addAnswer(conceptanswer);
+		conceptService.saveConcept(concept);
+		Assert.assertNotNull(concept.getDateChanged());
+		Date date=concept.getDateChanged();
+
+		Assert.assertTrue(concept.removeAnswer(conceptanswer));
 		conceptService.saveConcept(concept);
-		
 		Assert.assertNotNull(concept.getDateChanged());
+		Date date1=concept.getDateChanged();
+		Assert.assertFalse(date.equals(date1));
 		Assert.assertNotNull(concept.getChangedBy());
```

Comment:
@wluyima Audit info is set even if an item is added to concept,so I'm comparing Date objects just to make sure that audit info is set when an item is removed.